src/components/signIn/SignIn.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PopupWithForm from '../popupWithForm/PopupWithForm';
+
+function SignIn(props) {
+  return(
+    <PopupWithForm name="sign-in" title="Sign in" isOpen={props.isOpen} onClose={props.onClose} isLoading={props.isLoading}>
+
+      <div className="popup__input-wrapper">
+        <label className="popup__input-label" for="email-input">Email</label>
```

Comment:
the console said these ought to be `htmlFor`

Reply:
Changed these too